### ISSUE TITLE
[Accessibility] Missing focus experience on buttons after mouse click

### ISSUE THREAD

**Issue by vaibhav1192** [OfficeDev/office-ui-fabric-react#1543]
### Bug Report

- __Package version(s)__: 2.10.4

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? No

Requested priority: Normal

#### Describe the issue:

There is no focus experience for the command button.

Repro steps:-
1. Click the button using mouse
2. Move away the cursor from the button.
3. Now there is no focus exp for button.

#### Expected behavior:

Since button is still in focus. There should be some focus experience. Something like black or blue border.

### If applicable, please provide a codepen repro:

Click any button and then move away on http://codepen.io/dzearing/pen/MJBGog

**Comment by vaibhav1192** [comment]
Looks like this is the right behavior.
